### PR TITLE
[COC-26] Added Recipe Detail Screen

### PR DESCRIPTION
**Pull request by Alaple**
# Description

We added the Recipe Detailed screen. For this we added a xml, fragment and ViewModel files.

# Image evidence

image

### CODE REVIEW

File: app/src/main/java/com/bifrost/cocinarte/fragments/RecipeDetailFragment.kt [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        txtTitle.setText("DESCRIPTION")
+        txtDescription.setText("Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.")
+        btnPrepare.setText("PREPARE")
+    }
+
+    private fun initializeButtons() {
+        // PREPARE button
+        btnPrepare.setOnClickListener() {
+            // TODO Prepare
+            Snackbar.make(rootLayout, "TODO PREPARE", Snackbar.LENGTH_SHORT).show()
```
Comment (manuelfalonso) [inline app/src/main/java/com/bifrost/cocinarte/fragments/RecipeDetailFragment.kt:68]: Despues agregalo al backlog como una tarea que falta hacer. asi no nos colgamos con este TODO

**Review by manuelfalonso — APPROVED** [review]
image

**Review by JonathanJusid — APPROVED** [review]
ok!

File: app/src/main/java/com/bifrost/cocinarte/fragments/RecipeDetailFragment.kt (hunk 1 above)
Comment (Alaple) [reply app/src/main/java/com/bifrost/cocinarte/fragments/RecipeDetailFragment.kt:68]: Thanks! Already added to the Jira backlog, check here: 

https://pf-ort-grupoc.atlassian.net/browse/COC-63?atlOrigin=eyJpIjoiNjE0NWMwYzJhMzEwNGQ5OGIyYTI3MzI2YzU4MDQ3MjEiLCJwIjoiaiJ9

**Pull request merged**